features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/Credentials/Credentials.jsx

```diff
@@ -32,6 +32,7 @@ import Application from 'AppData/Application';
 import SubscribeToApi from 'AppComponents/Shared/AppsAndKeys/SubscribeToApi';
 import { ScopeValidation, resourceMethods, resourcePaths } from 'AppComponents/Shared/ScopeValidation';
 import { ApiContext } from '../ApiContext';
+import AuthManager from '../../../../data/AuthManager';
```

Comment:
we can use import AuthManager from 'AppData/AuthManager';  instead?

Reply:
Fixed